packages/cli/src/runners/task-runner-process.ts

```diff
@@ -47,6 +47,7 @@ export class TaskRunnerProcess {
 		'PATH',
 		'NODE_FUNCTION_ALLOW_BUILTIN',
 		'NODE_FUNCTION_ALLOW_EXTERNAL',
+		'N8N_RUNNERS_MAX_CONCURRENCY',
```

Comment:
We should maybe make these more explicit, rather than a passthrough env, just in case we change/add a new configuration option down the line.

Reply:
Makes sense. Changed it to be passed explicitly